packaging/machine-learning-api.spec

```diff
@@ -0,0 +1,171 @@
+###########################################################################
+# Default features for Tizen releases
+# If you want to build RPM for other Linux distro, you may need to
+# touch these values for your needs.
+%define		enable_tizen_privilege 1
+%define		enable_tizen_feature 1
+
+###########################################################################
+# Conditional features for Tizen releases
+
+# Disable a few features for TV releases
+%if "%{?profile}" == "tv"
+%define		enable_tizen_privilege 0
+%endif
+
+# If it is tizen, we can export Tizen API packages.
+%bcond_with tizen
+
+# Note that debug packages generate an additional build and storage cost.
+# If you do not need debug packages, run '$ gbs build ... --define "_skip_debug_rpm 1"'.
+%if "%{?_skip_debug_rpm}" == "1"
+%global debug_package %{nil}
+%global __debug_install_post %{nil}
+%endif
+
+###########################################################################
+# Package / sub-package definitions
+Name:		capi-machine-learning-inference
+Summary:	Tizen native API for NNStreamer
+# Synchronize the version information with NNStreamer.
+# 1. Ubuntu : ./debian/changelog
+# 2. Tizen  : ./packaging/machine-learning-api.spec
+Version:	1.7.1
```

Comment:
After the first release, you may increase version numbers independently from nnstreamer. (as long as it is higher than 1.7.1)

Reply:
I changed version info (0.0.1) as this is first commit for API repo.

Reply:
No. You must have a version that is higher than 1.7.1 (I suggest 1.7.2 or 2.0.0). Otherwise, Tizen OBS will obsolete your package.

Reply:
Updated to v1.7.2